R/convertFromFlowCore.R

```diff
@@ -0,0 +1,94 @@
+library("stats")
+
+#' Convert a flowCore object to a CellEngine object
+#'
+#' Creates a CellEngine object from a flowCore object, saving it to CellEngine.
+#'
+#' @param flowObject A flowDensity object or a flowCore Gate
+#' @param experimentId The ID of the experiment for which to apply the ScaleSet.
+#' @param name The name of the CellEngine gate. If not set, it will default to the
+#' \code{filterId} of the flowCore object.
+#' @param ... Other parameters passed to \code{createRectangleGate}, \code{createPolygonGate}, etc.
+#' @export
+#' @examples
+#' \dontrun{
+#' sngl <- flowDensity(file, params)
+#' exp <- getExperiment("5d2f8b4b21fd0676fb3a6a8c")
+#' fromFlowCore(sngl, exp$`_id`, name = "my gate")
+#' }
+fromFlowCore <- function(flowObject, experimentId, name = NULL, ...) {
+  if (is.null(name)) {
+    name <- flowObject@filterId
```

Comment:
flowCore has a generic function for this that will work with transformLists and other types in the future:
```suggestion
    name <- identifier(flowObject)
```

Reply:
nice!